source/macos/ADCMacOSVisualizer/ADCMacOSVisualizer/RootViewController.swift

```diff
@@ -123,6 +125,13 @@ class RootViewController: NSViewController, NSTableViewDelegate, NSTableViewData
         self.buttonConfig = buttonConfig
     }
     
+    private func setupInputFieldConfig() {
+        if darkTheme {
+            self.inputFieldConfig = .init(height: 26, leftPadding: 8, rightPadding: 8, yPadding: 0, focusRingCornerRadius: 8, borderWidth: 0.3, wantsClearButton: true, fontSize: 14, highlightedColor: .init(red: 1, green: 1, blue: 1, alpha: 0.11), backgroundColor: .init(red: 0.148, green: 0.148, blue: 0.148, alpha: 1))
+        } else {
```

Comment:
Minor: In general as a practice, we can avoid using `.init` and use the instance name for better readability.
In all places in this PR